build-tools/packages/build-tools/src/fluidBuild/tasks/workers/workerPool.ts

```diff
@@ -3,10 +3,11 @@
  * Licensed under the MIT License.
  */
 
-import { ChildProcess, fork } from "child_process";
-import { EventEmitter } from "events";
-import { Readable } from "stream";
-import { Worker } from "worker_threads";
+import { ChildProcess, fork } from "node:child_process";
```

Comment:
why did we have to add `node:` to the import?

Reply:
It's not required, but it's best practice and helps avoid resolution problems like we recently had with events. This is an example of "clean things up when you touch the code."

Reply:
It is required by our stricter eslint configs though :)